### ISSUE TITLE
about hdr results

### ISSUE THREAD

**Issue by fancyicookie** [chxy95/HDRUNet#4]
Hello, I read your paper and have some problem. Can I ask you some question?
Obviously, the real HDR looks like more dark, so the HDR results need to be processed by tone-mapping for visualization. But everyone can use different tone-mapping methods to see it. And different methods will display different results, aren't they?
If I have a HDR display, can I see the HDR image directly without using tone-mapping?

**Comment by chxy95** [comment]
@fancyicookie No. This repo are designed for the data provided by the challenge, which are not encoded in HDR format. So the HDR images should be processed by tone-mapping for visualization. For fair comparison, you should use the same tone-mapping method for different algorithms. We use the official method, μ-law tone-mapping. 

In fact, as far as I know, there is no HDR image format that is designed for consumer-grade HDR displays. The current HDR image formats, such as '.hdr', '.exr' and '.tiff', are not display format. Besides,  the common HDR standard, such as HDR10 and HLG, are designed for video, which involves video encoding&decoding.  Actually, we also have a work submitted to ICCV2021 for generating content encoded in HDR standard (i.e. the content can be shown on HDR display). We will release related codes after the ICCV result come out.

**Comment by fancyicookie** [comment]
Ok，thank you for your reply very much!!
